tests/utils/verification/volumeproperties.go

```diff
@@ -55,42 +55,37 @@ func GetVMAttachedToVolUsingAdminCli(volName string, hostname string) string {
 	return volProps[1]
 }
 
-// GetVolumePropertiesAdminCli returns capacity, attached-to-vm and disk-format field
-// for volume using Admin cli
-func GetVolumePropertiesAdminCli(volName string, hostname string) string {
-	cmd := admincli.ListVolumes + "-c volume,attached-to,capacity,disk-format 2>/dev/null | grep " + volName
-	op, _ := ssh.InvokeCommand(hostname, cmd)
-	if op == "" {
-		log.Fatal("Null value is returned by admin cli when looking for, size, disk-format and attached to vm. Output: ", op)
-	}
-	if len(strings.Fields(op)) != 4 {
-		log.Fatalf("Output is expected to consist of four elements only - "+
-			"volume name, attached-to-vm status, size and disk-format. Actual output: %s", op)
+// GetPropertiesForAllVolumesAdminCli returns a map of all the volumes from ESX and their corresponding
+// properties returned - capacity, attached-to-vm and disk-format
+func GetPropertiesForAllVolumesAdminCli(hostName string) map[string][]string {
```

Comment:
The method name is too big .. it should be simplified .. how about his `GetAllVolProperties` and the method doc mentions how are we retrieving.